doc/fgpu.design.rst

```diff
@@ -450,12 +450,22 @@ floating-point precision when accumulating a large number of samples.
 
 Network transmit
 ----------------
-The current transmit system is quite simple. A single spead2 stream is created,
-with one substream per multicast destination. For each output chunk, memory
-together with a set of heaps is created in advance. The heaps are carefully
-constructed so that they reference numpy arrays (including for the timestamps),
-rather than copying data into spead2. This allows heaps to be recycled for new
-data without having to create new heap objects.
+The current transmit system is quite simple. By default a single spead2 stream
+is created, with one substream per multicast destination. For each output
+chunk, memory together with a set of heaps is created in advance. The heaps are
+carefully constructed so that they reference numpy arrays (including for the
+timestamps), rather than copying data into spead2. This allows heaps to be
+recycled for new data without having to create new heap objects.
+
+If the traffic for a single engine exceeds the bandwidth of the network
+interface, it is necessary to distribute it over multiple interfaces. In this
+case, several spead2 streams are created (one per interface). Each of them has
+a substream for every multicast destination, but they are not all used (the
+duplication simplifies indexing). When heaps are transmitted, a stream is
+selected for each heap to balance the load. Descriptors and stop heaps are
```

Comment:
> When heaps are transmitted, a stream is selected for each heap to balance the load.

While this is technically true, I've rather been reading `_multi_send` logic as 'alternate/round-robin across all streams available for sending'. Not saying you should change it; perhaps your wording is simpler (principle of least astronishment and all that).

Reply:
I didn't want the design doc to get into the minutiae of how the selection is done.